tumpasrc/__init__.py

```diff
@@ -42,9 +42,8 @@ class PasswordEdit(QtWidgets.QLineEdit):
     }
     """
 
-    def __init__(self, parent):
-        self.parent = parent
-        super().__init__(self.parent)
+    def __init__(self):
+        super().__init__()
```

Comment:
So the `parent` argument is optional. On sending the `parent` argument, it attaches to the widget at that moment. Since both the name and email field were created with null parents, and then added to the widget, but in case of the PasswordEdit, because of the parent argument, it was attached to the widget on creation itself, so technically the first to be attached to the widget was the PasswordEdit field. Hence the tab order was wrong.

Given we are creating all the widgets with null parents, so I removed the parent argument here as well, to have the same behaviour through out